### DISCUSSION TITLE
How to supply non-linguistic symbols file

### DISCUSSION THREAD

**Discussion by feddybear** [espnet/espnet#3625] [Q&A] [answered]
Hi, as the title suggests, I was wondering how the format of non-linguistic symbols file should be?  I couldn't find any info on the scripts nor from the documentation.
Is it per line, comma separated, etc.?

Thank you.

**Comment by sw005320** [comment, marked as answer]
There is no format since each corpus uses a different way of specifying it.
We often use the bracket, e.g., `<noise>`, and some other corpus uses `[NOISE]` but any symbol that would not conflict with the token is fine.
Then, during our process, you can just make a list and pass it to the recipe by `--nlsyms_txt`.
This would be
```
$ cat non_lang_syms.txt
<noise>
<laugh>
```
something like that.

You can also check some other recipes, e.g., WSJ
https://github.com/espnet/espnet/blob/master/egs2/wsj/asr1/run.sh#L17

**Reply by feddybear** [reply]
Thank you very much, Watanabe-san!
